content/blogposts/2024/I-welcome-our-self-driving-ide-overlords.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "I, for one, welcome our new Self-Driving IDE overlords"
```

Comment:
Please change the title (and markdown filename!) to "The Self-Driving IDE is Coming"

Thank you!